test/metadata/generated/BodyComplex/package.json

```diff
@@ -32,7 +35,7 @@
     "url": "https://github.com/azure/azure-sdk-for-js/issues"
   },
   "scripts": {
-    "build": "tsc && tsc -p tsconfig.esm.json && webpack",
+    "build": "tsc && rollup -c rollup.config.js && uglifyjs ./dist/bundle.js -c -m -o ./dist/bundle.min.js",
```

Comment:
Does it provide enough value for the browser use cases to include a .min.js within the package?

Reply:
Yes, this is a must. Customers using CDNs like jsdelivr etc. which make the npm registry available as a CDN will appreciate minified assets (even though if they really care about code size they should bundle).

Reply:
Makes sense--we will definitely stick with shipping the .min.js